webapp/src/adapters/solid/MapPointManager.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    buildThing,
+    createSolidDataset,
+    createThing,
+    getPodUrlAll, getSolidDataset,
+    saveSolidDatasetAt,
+    setThing,
+} from "@inrupt/solid-client";
+import {SCHEMA_INRUPT} from "@inrupt/vocab-common-rdf";
+import MapPoint from "../../domain/MapPoint";
+import SolidSessionManager from "./SolidSessionManager";
+
+export default class MapPointManager {
+
+    private readonly MAPPOINTSURL: string = "LoMap/MapPoints";
+    private sessionManager: SolidSessionManager = SolidSessionManager.getManager();
+
+    /**
+     * It stores a MapPoint in the POD of the current user logged in
+     *
+     * */
+    public async createNewMapPoint(newLocation: MapPoint): Promise<void> {
+        const podURL = await getPodUrlAll(this.sessionManager.getWebID(), {fetch: fetch});
+        try{
+            await this.appendToDataSet(podURL[0] + "public/" + this.MAPPOINTSURL, newLocation);
```

Comment:
I would put the LoMap folder at the same level as /public

Reply:
Ok we could consider that tomorrow in the meeting